src/main/python/pgshovel/replication/streams/kafka.py

```diff
@@ -29,7 +43,7 @@ def consume(self, state):
         # hard we don't really need to deal with it right now.
         assert len(consumer.offsets) is 1
 
-        if state.HasField('stream_state'):
+        if state.stream_state.HasField('consumer_state'):
```

Comment:
Good point, this makes sense in case this object initialized in memory, but not actually validated in memory — I was assuming that both fields existed, since they are both required: https://github.com/disqus/pgshovel/blob/1f18a7b75494bdeabd8f78b872f50560a91f9951//src/main/protobuf/pgshovel/interfaces/replication.proto#L70-L76

I guess `stream_state` would just be the default value now (no `consumer_state`) which is why this works?